src/helpers/utils.ts

```diff
@@ -246,3 +246,19 @@ export function handleStatusRedirect(
     else location.href = '/consent_osp'
   } else history.push('/landing')
 }
+
+export const isObjectEmptyOrFalsy = (obj) => {
+  // Check if the object is empty
+  if (Object.keys(obj).length === 0) {
```

Comment:
How about cover undefined, etc. with

`if (!obj || Object.keys(obj).length === 0) {`

that way we can write `isNextBtnDisabled` more compact

Reply:
Agreed 👍